supplierProduct/tests.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+from supplierProduct.models import supplierProduct, Product, Supplier
+
+supID0 = 123
+supID1 = 222
+supID2 = 127
```

Comment:
Constants should be in upper case, but it might be even better to have it as a fixture, so you know that if one test changed one of those, it's loaded with the original content for the next test.